contributing.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: article
+title: Contributing Guide
+---
+
+This wiki is cool, right? But you know what would make it even better? Your help! This site was started by students just like you, and we need your help in keeping it up-to-date and adding new content.
+
+* TOC
+{:toc}
+
+## Make a Github Account
+
+Already have one? Skip down to either [Editing a Page](#editing-a-page) or [Making a New Page](#making-a-new-page).
+
+## Editing a Page
+
+If you've heard about Git or Github and how difficult it can't be, don't worry. Github makes it easy with a web interface. Just visit our [repository](https://github.com/CornellCSWiki/testwiki) and find the page you want to edit. Then just hit the edit button!
+
+<!-- TODO: add screenshot of the button. How do I/can I do this from the web interface? -->
```

Comment:
Don't think you can do this: either the image has to go in the repository (can this be done from the web interface?), or hosted externally.